scripts/docker-compose.yml

```diff
@@ -20,13 +20,16 @@ services:
     tty: true
     depends_on:
       - dc
+      - db
     environment:
       AGENT_DB_HOST: 'db'
       AGENT_DB_USER: 'agent'
       AGENT_DB_PASSWORD: 'agent'
       STREAMSETS_URL: 'http://dc:18630'
-#    ports:
-#    - 8080:80
+    ports:
+    - 8080:80
```

Comment:
Spaces are missing :)
Why did you uncomment it?

Reply:
So user when installing the agent will have an agent api available